src/containers/FacilityForm.js

```diff
@@ -74,53 +76,62 @@ function FacilityForm({ backend, history }) {
     );
   }
 
-  return (
-    <Form
-      buttonLabel="Submit"
-      onSubmit={handleSubmit}
-      title="Add a new facility"
-      disabled={!Object.keys(requiredFields).every((key) => !!fields[key])}
-      description="Enter some information about your facility."
-    >
-      <InputText
-        label="Name of the facility"
-        value={dropSiteFacilityName}
-        customOnChange={handleFieldChange('dropSiteFacilityName')}
-      />
-      <InputText
-        label="Zip code"
-        value={dropSiteZip}
-        customOnChange={handleFieldChange('dropSiteZip')}
-      />
-      <div css={{ display: 'flex', '> *': { width: '50%' } }}>
-        <InputText
-          label="City"
-          value={dropSiteCity}
-          customOnChange={handleFieldChange('dropSiteCity')}
-        />
-        <InputDropdown
-          placeholder="State"
-          value={dropSiteState}
-          options={states}
-          customOnChange={handleFieldChange('dropSiteState')}
-        />
-      </div>
-      <InputText
-        label="Full Street Address"
-        value={dropSiteAddress}
-        customOnChange={handleFieldChange('dropSiteAddress')}
-      />
-      <InputText
-        label="Website URL (optional)"
-        value={dropSiteUrl}
-        customOnChange={handleFieldChange('dropSiteUrl')}
-      />
-      <Note>
+  const formData = {
```

Comment:
I considered putting this in a data file (eg: `/containers/FacilityForm/data.js`), but we would need to pass a number of values to that data file that are available for free in this file (eg: customOnChange functions and value).

We could split the data into two types -- static and dynamic -- with the static data living in the data file (labels, types, etc) and dynamic data that comes from state and local methods can live in this file.

I'm open to ideas!